pseudopotential_calculator/calculations/slab.py

```diff
@@ -101,80 +162,23 @@ def get_surface(
     n_fixed_layer: int = 0,
     n_free_layer: int = 0,
     n_vacuum_layer: int = 0,
+    adsorbate_height: float = 0,
```

Comment:
what if I don't want an adsorbate on the surface?

Reply:
so no adsorbate_height is provided and therefore there are no extra layers of vacuum added.